src/app/services/offre-de-stage/offre-de-stage.service.ts

```diff
@@ -27,9 +27,9 @@ export class OffreDeStageService {
   }
 
   getOffreDeStage(
-    _id: string
+    _id?: string
```

Comment:
On ne veut pas que `_id` soir `undefined` quand on appelle la fonction. 

```suggestion
    _id: string
```